pkg/crawl/crawler.go

```diff
@@ -36,16 +36,18 @@ type Crawler struct {
 
 type Stats struct {
 	// Total number of files scanned.
-	FilesScanned int64 `json:"filesScanned"`
+	FilesScanned uint64 `json:"filesScanned"`
 	// Number of paths that were not considered due to "permission denied" errors
-	PermissionDeniedCount int64 `json:"permissionDeniedErrors"`
+	PermissionDeniedCount uint64 `json:"permissionDeniedErrors"`
 	// Number of paths that were attempted to be processed but encountered errors.
-	PathErrorCount int64 `json:"pathErrors"`
+	PathErrorCount uint64 `json:"pathErrors"`
+	// Number of paths that were skipped due to config/size limits
+	PathSkippedCount uint64 `json:"pathsSkipped"`
 }
 
 // MatchFunc is used to match a file for processing.
 // If returning a positive finding, a file will be passed onto the ProcessFunc.
-type MatchFunc func(ctx context.Context, path string, d fs.DirEntry) (Finding, Versions, error)
+type MatchFunc func(ctx context.Context, path string, d fs.DirEntry) (Finding, Versions, uint64, error)
```

Comment:
Please add a comment for what this `uint64` should be, it's not clear from the signature.

Reply:
I think this was from https://github.com/palantir/log4j-sniffer/pull/64 and so will not need any actions within this PR